code/modules/antagonists/clockcult/clock_structures/ark_of_the_clockwork_justicar.dm

```diff
@@ -81,7 +81,7 @@
 	priority_announce("Massive [Gibberish("bluespace", 100)] anomaly detected on all frequencies. All crew are directed to \
 	@!$, [text2ratvar("PURGE ALL UNTRUTHS")] <&. the anomalies and destroy their source to prevent further damage to corporate property. This is \
 	not a drill.", "Central Command Higher Dimensional Affairs", 'sound/magic/clockwork/ark_activation_sequence.ogg')
-	set_security_level("delta")
+	set_security_level("red")
```

Comment:
Stationending event, should stay delta alert to represent it's *important* to destroy that gateway.